services/repository-managers/src/main/java/org/sagebionetworks/repo/manager/team/TeamManagerImpl.java

```diff
@@ -335,7 +335,8 @@ private void bindTeamName(String name, Long teamId){
 
 	@Override
 	public PaginatedResults<TeamMember> listMembers(String teamId, TeamMemberTypeFilterOptions memberType, long limit,
-			long offset) throws DatastoreException {
+			long offset) throws DatastoreException, NotFoundException {
+		get(teamId);
```

Comment:
Could be a better approach to create a dedicated validation method that takes in input the id of the team and checks its existence, maybe without fetching the whole team data from the database.